MvvmCross-AndroidSupport/MvvmCross.Droid.Support.V7.AppCompat/MvxCachingFragmentCompatActivity.cs

```diff
@@ -210,6 +210,13 @@ protected virtual void ShowFragment(string tag, int contentId, Bundle bundle, bo
 			{
 				(fragInfo.CachedFragment as Android.Support.V4.App.Fragment).Arguments.Clear();
 				(fragInfo.CachedFragment as Android.Support.V4.App.Fragment).Arguments.PutAll(bundle);
+
+                var childViewModelCache = Mvx.GetSingleton<IMvxChildViewModelCache>();
+                if(childViewModelCache.Exists(fragInfo.CachedFragment.ViewModel.GetType()))
+                {
+                    fragInfo.CachedFragment.ViewModel = childViewModelCache.Get(fragInfo.CachedFragment.ViewModel.GetType());
+                    childViewModelCache.Remove(fragInfo.CachedFragment.ViewModel.GetType());
```

Comment:
Same here.  Also... is there a way to consolidate this code since it's the same as the non support version?